src/testUtil/TestCaseRecorder.ts

```diff
@@ -243,15 +261,14 @@ export class TestCaseRecorder {
   }
 
   async postCommandHook(returnValue: any) {
-    if (this.testCase == null) {
+    if (this.testCase == null && !this.captureErrorTests) {
```

Comment:
As implied above, I'd just let error handler do an `await this.finishTestCase();`, then this function can remain unchanged i believe